examples/iris_testbed/tests/hdlc_txvr/python_hdlc/hdlc.py

```diff
@@ -135,7 +143,24 @@ def hdlc_tx(self, ser):
 		# global send_seq_no, hdlcOutbox, mutex
 	
 		while True:
-			mail = self.hdlcOutbox.get(block=True) # here we can use blocking to see if we have anything to send (will be removed automatically from the queue)
+			with self.mutex:
+				if self.uart_lock == True:
+					timeout = last_sent + HDLC_RETRANS_TIMEO_USEC/1000000 - time() 
+					if timeout < 0: # if we haven't got ACK back
+						newMail = hdlcmsg(HDLCMT.HDLC_MSG_RESEND, self.sender_msg, self.sender_id, self.sender_mbox)	
+						#TODO
+						try:
+							self.hdlcOutbox.put(newMail, block=False)
+						except queue.Full as e:
+							print ("hdlc: no space in hdlc thread outbox. ERROR!!\n")
+							ser.close()
+							exit(0)
+
+						mail = self.hdlcOutbox.get(block=True) # here we can use blocking to see if we have anything to send (will be removed automatically from the queue)
```

Comment:
Try to stick with one naming convention in all your files. That is, use either camelCase or under_scoring. Typically speaking, class names will have no underscores and use capital letters like MyClass. Variables and methods will use underscores like MyClass.this_var or MyClass.my_func().